exporters/user_events/src/recordable.cc

```diff
@@ -17,50 +17,58 @@ namespace logs
 
 namespace api_logs = opentelemetry::logs;
 
-Recordable::Recordable() noexcept
-{
-  event_builder_.Reset("OpenTelemetry-Logs");
-
-  utils::PopulateAttribute("__csver__", static_cast<uint16_t>(0x400), event_builder_);
-}
+Recordable::Recordable() noexcept {}
 
 void Recordable::SetSeverity(api_logs::Severity severity) noexcept
 {
   uint8_t severity_value = static_cast<uint8_t>(severity);
-  if (severity_value == 0 || severity_value > 24)
+  if (severity_value > 24)
   {
     OTEL_INTERNAL_LOG_ERROR(
         "[user_events Log Exporter] Recordable: invalid severity value: " << severity_value);
-    severity_value = 1;
+    severity_value = 0;
   }
 
+  severity_    = severity_value;
   level_index_ = (severity_value - 1) >> 2;
-
-  cs_part_b_bookmark_size_ += 2;
-  event_builder_.AddValue("severityNumber", static_cast<uint16_t>(severity_value),
-                          event_field_format_default);
-  auto severity_text = api_logs::SeverityNumToText[static_cast<uint32_t>(severity_value)].data();
-  event_builder_.AddString<char>("severityText", severity_text, event_field_format_default);
 }
 
 void Recordable::SetBody(const opentelemetry::common::AttributeValue &message) noexcept
 {
-  // Set intial bookmark size to 1 for body below.
-  cs_part_b_bookmark_size_++;
+  if (severity_ == 0)
+  {
+    OTEL_INTERNAL_LOG_ERROR("[user_events Log Exporter] Recordable: severity is not set.");
+    return;
+  }
+
+  auto event_name = !event_name_.empty() ? event_name_.data() : "OpenTelemetryLogs";
```

Comment:
Rust uses default as "Logs". Either is fine, as long as we are consistent. cc @cijothomas.